code/modules/projectiles/magazines/revolvers.dm

```diff
@@ -24,6 +24,7 @@
 
 /obj/item/ammo_magazine/revolver/heavy
 	name = "\improper M44 heavy speed loader (.44)"
+	desc = "A revolver speed loader containing heavy bullets. While less heavy hitting than the traditional roundsS, these are more accurate."
```

Comment:
rounds has a double S on the end

Probably use "While less damaging than traditional rounds" for conciseness too